api/resource/definitions/cluster/cluster.proto

```diff
@@ -3,6 +3,7 @@ syntax = "proto3";
 package talos.resource.definitions.cluster;
 
 option go_package = "github.com/siderolabs/talos/pkg/machinery/api/resource/definitions/cluster";
+option java_package = "dev.talos.api.resource.definitions.cluster";
```

Comment:
the files under `api/resource/definitions` are automatically generated (you can check that by running `make generate`), so you'd need to update the generator (should be hack/structprotogen iirc)

Reply:
Added and generated!